tasks/rh-sign-image-cosign/tests/mocks.sh

```diff
@@ -137,14 +137,25 @@ function skopeo() {
   fi
 }
 function cosign () {
-  echo "$@" >> $(workspaces.data.path)/mock_cosign_calls
-  echo "running cosign: $@"
-  # mock cosign failing the first 3 calls for the retry test
-  if [[ "$@" == *":retry-tag"* ]]
-  then
-    if [[ $(cat $(workspaces.data.path)/mock_cosign_calls | wc -l) -le 3 ]]
-    then
-      echo "expected cosign call failure for retry test"
+  # check if call should end successfully
+  successfull_run=$(sed -n '1p' $(workspaces.data.path)/mock_cosign_success_calls && \
+    sed -i '1d' $(workspaces.data.path)/mock_cosign_success_calls)
+
+  if [ "$1" = "verify" ]; then
+    mock_existing_sig_file=$(echo "${*: -1}" | tr "/" "-")
```

Comment:
My guess is that you don't have the `"running cosign"` echo here to avoid spoiling the output, but you can do `echo >&2` and that way it will not have any negative effect.

Reply:
but here it doesn't matter

Reply:
It's just that you would see the calls directly in the test log. But up to you.